curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sum-all-primes.english.md

```diff
@@ -8,9 +8,14 @@ forumTopicId: 16085
 
 ## Description
 <section id='description'>
-Sum all the prime numbers up to and including the provided number.
-A prime number is defined as a number greater than one and having only two divisors, one and itself. For example, 2 is a prime number because it's only divisible by one and two.
-The provided number may not be a prime.
+
+A prime number is a whole number greater than 1 with exactly two divisors: 1 and
+itself. For example, 2 is a prime number because it is only divisible by 1 and 2. In
+contrast, 4 is not prime since it is divisible by 1, 2 and 4.
+
+Rewrite sumPrimes to return the sum of all prime numbers that are less than or
```

Comment:
```suggestion
Rewrite `sumPrimes` to return the sum of all prime numbers that are less than or
```
Also, could you put this and the 'Read-search-ask' bit in the instructions section?

Reply:
I left everything in the description area to keep it consistent with how all the other algorithms are. I think if we want to move it, we should make an issue to move all of them.